### ISSUE TITLE
Hot reloading express server

### ISSUE THREAD

**Issue by somus** [Hashnode/mern-starter#153]
https://github.com/ericclemmons/webpack-hot-server-example

**Comment by carloscuatin** [comment]
remove nodemon ?

**Comment by somus** [comment]
Yes, once the express server is hot reloaded, we will not need nodemon.

**Comment by carloscuatin** [comment]
👍 excelent

**Comment by SOSANA** [comment]
This is cool!!!

**Comment by SOSANA** [comment]
you think this will improve performance over nodemon?

**Comment by koshuang** [comment]
Great! If the server can be hot reloaded, to develop api will be much faster!

**Comment by DraconPern** [comment]
I think this issue can be closed since I see HMR working.

**Comment by kamiranoff** [comment]
Anyone knows how to implement it ?

**Comment by qlqllu** [comment]
is anyone working on this?

**Comment by u0078867** [comment]
I played with this, very cool: https://github.com/glenjamin/ultimate-hot-reloading-example
